### ISSUE TITLE
More modeling options

### ISSUE THREAD

**Issue by ndiamant** [broadinstitute/ml4h#47]
I want to add more options to `make_multimodal_to_multilabel_model`
- [x] [Dilated convolution](https://arxiv.org/abs/1803.01271)
- [x] [More normalization schemes](https://arxiv.org/pdf/1803.08494.pdf)
- [x] Optimizer options
- [x] Learning rate schemes (cosine annealing)
- [x] VAE (with supervision)
- [x] [Triplet loss](https://omoindrot.github.io/triplet-loss) to improve embeddings
- [ ] Quantization to train those brains
- [ ] Infer multiple models at the same time for ensembling
